controllers/devopsconfig/agent_config_template.go

```diff
@@ -0,0 +1,59 @@
+package devopsconfig
+
+import (
+	"context"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1core "k8s.io/client-go/kubernetes/typed/core/v1"
+	"k8s.io/klog"
+)
+
+const (
+	templateConfigMapName = "jenkins-casc-config-template"
+	templateConfigKey     = "jenkins.yaml"
+)
+
+// Get jenkins casc configuration template
+func getJenkinsCascConfigTemplate(c *Controller, namespace string, template string) (string, error) {
```

Comment:
Casc means Configration as Code. So I prefer to change the function name from `getJenkinsCascConfigTemplate` to `getJenkinsCascTemplate`.

Reply:
I prefer to use a Golang suggested comment style. See also https://blog.golang.org/godoc

Notice this comment is a complete sentence that begins with the name of the element it describes ...